Makefile

```diff
@@ -92,6 +92,10 @@ JWT_APP_ID ?= ""
 # golangci-lint
 GOLANGCI_LINT_VERSION := v1.54.2
 
+# The module import path varies stating with version 2
+MODULE_VERSION = $(shell if [[ "$(VERSION)" =~ 2\..* ]]; then echo "/v2"; else echo ""; fi)
```

Comment:
suggestion: let's move this into `scripts`.

question: Is this going to work for versions higher than 2? We should avoid remembering bumping this here whenever we do a major release.

Reply:
as an after-thought: Why would the Makefile need to be aware of older release versions? 🤔

Reply:
The reason we use the new makefile for older builds is to allow for fast builds on versions that did not know how to make them happen. That combined with the version 2 bump mixed very badly.
I think I should follow up with a proper fix for this that does not require checking out new code into an older branch, but that should be a separate PR. For this one I would document and move that to a script as suggested.
As for v3, I plan to have a better solution in place in the next days, so we do not have to update the script to version 3 and instead we can delete such script.

Reply:
I have sent a change moving this logic to a script and making it work for any version. But I still plan to follow up with a better fix that does not overwrite the old branch Makefile.

Reply:
@josvazg your plan sounds great, thank you! 🙌